build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2023.3.2"
```

Comment:
In the future you don't have to bump the GradleRIO version. I do that directly to the main repo so just rebase the change onto your fork.